packages/host/src/components/Login/Login.tsx

```diff
@@ -87,30 +151,64 @@ export const Login: React.FC = ({}) => {
         display="flex"
       >
         <Box style={{ width: 285 }}>
-          <Stack spacing={5}>
-            <Box display="flex" justifyContent="center">
-              <LoginIcon />
-            </Box>
-            <LoginTextInput fullWidth label={t('heading.username')} />
-            <LoginTextInput
-              fullWidth
-              label={t('heading.password')}
-              type="password"
-            />
-            <Autocomplete
-              renderInput={StoreAutocompleteInput}
-              options={stores}
-            />
-            <Box display="flex" justifyContent="flex-end">
-              <BaseButton
-                onClick={onLogin}
-                variant="outlined"
-                endIcon={<ArrowRightIcon />}
-              >
-                {t('button.login')}
-              </BaseButton>
-            </Box>
-          </Stack>
+          <form onSubmit={onLogin}>
+            <Stack spacing={5}>
+              <Box display="flex" justifyContent="center">
+                <LoginIcon />
+              </Box>
+              <LoginTextInput
+                fullWidth
+                label={t('heading.username')}
+                value={username}
+                onChange={e => setUsername(e.target.value)}
+                inputProps={{
+                  autoComplete: 'username',
+                }}
+              />
+              <LoginTextInput
+                fullWidth
+                label={t('heading.password')}
+                type="password"
+                value={password}
+                onChange={e => setPassword(e.target.value)}
+                inputProps={{
+                  autoComplete: 'current-password',
```

Comment:
I dunno.. chrome said this was important 🤷  same as adding the `<form/>` - that helps the password savers to work